client/overview/connection-sucess-notice.tsx

```diff
@@ -40,46 +40,25 @@ const ConnectionSuccessNotice: React.FC = () => {
 			/>
 		);
 	};
-
-	return ! isDismissed && ! onboardingTestMode ? (
+	const isPoDisabledOrCompleted = ! ( isPoEnabled && ! isPoComplete );
```

Comment:
Nitpicking. Given the name of the variable is disabled or completed, I suggest we do the same in the definition:
```suggestion
	const isPoDisabledOrCompleted = ! isPoEnabled || isPoComplete;
```

Reply:
changed in https://github.com/Automattic/woocommerce-payments/pull/8399/commits/2adf682ee866d42705654cc9ae8b1e0678d6eb02 . thx.